pwa/src/components/common/textareaGroup.tsx

```diff
@@ -0,0 +1,39 @@
+import * as _ from "lodash";
```

Comment:
Do we still want to include this in our `NL-Design-System` repository? 

This would mean:

1. Remove this code from this repository;
2. Add this code to the `NL-Design-System` repository;
3. Create a pull request there and eventually merge it (this merge will give you a new version number of the package);
4. Include this new version number in `package.json` within this repository;
5. Finally, include the TextareaGroup components from the package and no longer from the local file.